src/components/Tabber/Tabber.tsx

```diff
@@ -72,21 +101,22 @@ function Tabber(): ReactNode {
         <div className={scss.postModesContainer}>
           <PostModes
             currentPostModeId={tabs[currentTab].postModeOnView}
-            currentTab={tabs[currentTab].account}
+            currentTab={tabs[currentTab].account as Account}
             onChangePostMode={changePostMode}
           />
           <MainComposerBase
             accountId={tabs[currentTab].account.id.toString()}
             onChange={handleContentChange}
-            postMode={currentPostMode}
+            postMode={currentPostMode ?? undefined}
```

Comment:
Aqui já vai ser retornado PostMode ou undefined
```suggestion
            postMode={currentPostMode}
```